rfcs/0001-incentivization-transactions.md

```diff
@@ -0,0 +1,66 @@
+# Summary
+
+[summary]: #summary
+
+Provide a mechanism to pay users for having a verified identity and continuously adding data to their data host.
+
+# Motivation
+
+[motivation]: #motivation
+
+The protocol is deeply dependant on the network effects of users wanting applications and applications wanting users.
```

Comment:
I think "users wanting applications and applications wanting users" would be clearer as "data providers wanting data consumers, and data consumers wanting data providers". "Applications" is a bit context-free here

Reply:
Rephrased some.